cmd/lncli/cmd_open_channel.go

```diff
@@ -191,6 +196,7 @@ func openChannel(ctx *cli.Context) error {
 		MinConfs:         minConfs,
 		SpendUnconfirmed: minConfs == 0,
 		CloseAddress:     ctx.String("close_address"),
+		MaxValueInFlight: ctx.Int64("remote_max_value_in_flight_msat"),
```

Comment:
We strive to have each commit compile independently, so the RPC changes should come before this.